### PR TITLE
(MAINT) modulesync [067d08a]

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

**Comment by DavidS** [comment]
Testing ruby1.9 backwards compatibility. Please do not merge yet.

File: .gitignore [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
-spec/fixtures/modules/*
+spec/fixtures/
 .vagrant/
 .bundle/
 coverage/
 log/
 .idea/
 *.iml
-tmp/
+!spec/fixtures/
```
Comment (tphoney) [inline .gitignore]: this looks weird

Comment (DavidS) [inline .gitignore]: the only way to un-ignore the files in `spec/fixtures/modules`

Comment (tphoney) [inline .gitignore]: still weird. :)

**Pull request merged**